api-reference/echo-node-api/database-api.md

```diff
@@ -4627,9 +4629,47 @@ The public erc20 token data stored in the blockchain.
 }
 ```
 
+### check\_erc20\_token\(id\)
+
+Returns check on exist erc20 token.
```

Comment:
Return true if the contract exists and is ERC20 token contract registered by register_erc20_contract operation